phenotrex/transforms/annotation.py

```diff
@@ -27,17 +28,19 @@
     ('eggNOG5', 2),
 }
 
+
 class PreloadedProteinIterator(ProteinIterator):
     """Hack ProteinDataset to load from list directly."""
-    def __init__(self, protein_list: List[SeqRecord], aa_vocab, format):
+    def __init__(
+        self, protein_list: List[SeqRecord], aa_vocab, format
+    ):
         self.iterator = (x for x in protein_list)
         self.vocab = aa_vocab
         self.format = format
         self.start = 0
         self.pos = None
         self.step = 0
-        self.sequence = namedtuple('sequence',
-                                   ['index', 'id', 'string', 'encoded'])
+        self.sequence = namedtuple('sequence', ['index', 'id', 'string', 'encoded'])
```

Comment:
I refactored that to a typing.NamedTuple in `deepnog 1.2`, which allows for type annotations, like this:
```
sequence_tuple = NamedTuple('sequence',
                            [('index', int),
                             ('id', str),
                             ('string', str),
                             ('encoded', list),
                             ('label', torch.Tensor)])
```
Also, it's module-level now, so there is no more `self.sequence`.